modules/custom/az_core/css/az-bef-sidebar.css

```diff
@@ -0,0 +1,119 @@
+.az-bef-vertical ul {
+  list-style: none;
+}
+.az-bef-vertical .collapser::after {
+    font-family: "Material Icons Sharp";
+    -webkit-font-variant-ligatures: no-common-ligatures;
+    font-variant-ligatures: no-common-ligatures;
+}
+.az-bef-vertical .collapser {
+    top: 2px;
+    right: 20px;
+    font-size: 1rem;
+    display: inline-block;
+}
+
+.az-bef-vertical .collapser.level-0 {
+    top: 2px;
+    right: 20px;
+    font-size: 1.25rem;
+    display: inline-block;
+}
+.az-bef-vertical .collapser[aria-expanded=true]::after {
+  content: "remove";
+}
+.az-bef-vertical .collapser[aria-expanded=false]::after {
+  content: "add";
+}
+.az-bef-vertical .collapser.level-0[aria-expanded=true]::after {
+    content: "expand_less";
+}
+.az-bef-vertical .collapser.level-0[aria-expanded=false]::after {
+    content: "expand_more";
+}
+
+.az-bef-vertical {
+  margin-bottom: 1rem;
+}
+
```

Comment:
This can be replaced by mb-3 https://digital.arizona.edu/arizona-bootstrap/docs/2.0/utilities/spacing/

Reply:
@danahertzberg Is this important? Currently, this is being overridden by a `margin-bottom: 1.5rem` rule from AZ Barrio: https://github.com/az-digital/az_quickstart/blob/01cecb415f41c7f4b56b5a5e388772304bc6d786/themes/custom/az_barrio/css/style.css#L543-L547.